bentoml/handlers/dataframe_handler.py

```diff
@@ -66,6 +66,36 @@ def __init__(
         self.typ = typ
         self.input_columns = input_columns
 
+    def _get_type(self, item):
```

Comment:
I know it's probably less common in ML use cases, but would be nice to also add `boolean` and `date` type here? I think both OpenAPI spec and pandas dataframe have those